### ISSUE TITLE
Cyan fails to change app name in github actions

### ISSUE THREAD

**Issue by Cristiandis** [asdfzxcvbn/pyzule-rw#3]
https://github.com/Cristiandis/RevengeTweak/blob/pyzule-rw/.github/workflows/build.yml this is the action file
i was using pyzule before and it was working without issues, then i swapped to cyan and now it doesnt change the app name
here's: the command

cyan -n Revenge -k PyoncordIcon60x60@2x.png -d -i discord.ipa -o /Users/runner/work/RevengeTweak/RevengeTweak/Revenge.ipa -f /Users/runner/work/RevengeTweak/RevengeTweak/io.github.revenge-mod.app_0.0.1_iphoneos-arm.deb

everything else works except the app name

**Comment by Cristiandis** [comment]
full ci output
```Run cyan -n Revenge -k PyoncordIcon60x60@2x.png -d -i discord.ipa -o /Users/runner/work/RevengeTweak/RevengeTweak/Revenge.ipa -f /Users/runner/work/RevengeTweak/RevengeTweak/io.github.revenge-mod.app_0.0.1_iphoneos-arm.deb
[*] extracting ipa..
[*] extracted ipa
[*] extracted io.github.revenge-mod.app_0.0.1_iphoneos-arm.deb
[*] fixed dependency in RevengeTweak.dylib: /Library/Frameworks/Orion.framework/Orion -> @rpath/Orion.framework/Orion
[*] injected RevengeTweak.dylib
[*] injected RevengePatches.bundle
[*] auto-injected CydiaSubstrate.framework
[*] auto-injected Orion.framework
[*] restored entitlements
[*] changed name to "Revenge"
[*] updated app icon
[*] enabled documents support
[*] generating ipa with compression level 6..
[*] generated ipa at /Users/runner/work/RevengeTweak/RevengeTweak/Revenge.ipa
[*] deleting temporary directory..```

**Comment by asdfzxcvbn** [comment]
hey, im currently looking into this

im SUUUPer confused bc im looking at the code and it even saves, i dont know why this happens

**Comment by Cristiandis** [comment]
let me know when it gets fixed :)

**Comment by asdfzxcvbn** [comment]
just found the issue, im REALLLLLY stupid LMAO

update coming in a few minutes

**Comment by asdfzxcvbn** [comment]
fixed in v1.1.5: https://github.com/asdfzxcvbn/pyzule-rw/releases/tag/v1.1.5
